firebase-vscode/src/core/project.ts

```diff
@@ -74,48 +76,53 @@ export function registerProject({
     });
   });
 
-  broker.on("selectProject", async () => {
-    if (process.env.MONOSPACE_ENV) {
-      pluginLogger.debug(
-        "selectProject: found MONOSPACE_ENV, " +
-          "prompting user using external flow"
-      );
-      /**
-       * Monospace case: use Monospace flow
-       */
-      const monospaceExtension =
-        vscode.extensions.getExtension("google.monospace");
-      process.env.MONOSPACE_DAEMON_PORT =
-        monospaceExtension.exports.getMonospaceDaemonPort();
-      try {
-        const projectId = await selectProjectInMonospace({
-          projectRoot: currentOptions.value.cwd,
-          project: undefined,
-          isVSCE: true,
-        });
+  const command = vscode.commands.registerCommand(
```

Comment:
do we need to change the webview to call this command?

Reply:
The webview invokes the command through the HTMLBroker. Hence the "broker.on" at the end of the changes

Reply:
ah I missed that!